web/src/beta/lib/core/engines/Cesium/Feature/Box/index.tsx

```diff
@@ -86,16 +87,24 @@ const Box: React.FC<Props> = memo(function BoxPresenter({
             layerId={`${layerId}-edge${edge.isDraggable ? `-draggable` : ""}-${i}`}
             index={i}
             edge={edge}
-            isHovered={activeEdgeIndex === i || (!edge.isDraggable && !!activeBox)}
+            isHovered={
```

Comment:
We might need to handle here as well.
https://github.com/reearth/reearth/blob/fix/avoid_selecting_for_box/web/src/beta/lib/core/engines/Cesium/Feature/Box/index.tsx#L123